python/ray/experimental/sgd/pytorch/pytorch_runner.py

```diff
@@ -0,0 +1,163 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import logging
+import os
+import torch
+import torch.distributed as dist
```

Comment:
is this a standard abbreviation in pytorch?

Reply:
Yes, the do this in the [tutorial](https://pytorch.org/tutorials/intermediate/dist_tuto.html)